cmd/deploy/start.go

```diff
@@ -219,6 +220,8 @@ func WithLocalFile(cmd *cobra.Command, options *deployStartOptions, deployClient
 	if err != nil {
 		return nil, nil, errorUtils.NewWrappedError(ErrYAMLFileRead, err)
 	}
+	validationFailures := validate.Validate(file)
+	validate.LogValidationErrors(cmd.OutOrStdout(), validationFailures, false)
```

Comment:
just one question - there are various output types for the cli, does that take it into account? i guess some of the output types simply remove all output logging from the console.

Reply:
I'm just following how similar logging was done.